build-system/tasks/check-types.js

```diff
@@ -52,6 +57,76 @@ const getExtensionSrcPaths = () =>
  * @type {Object<string, Object|function():Object>}
  */
 const TYPE_CHECK_TARGETS = {
+  // Below are targets containing individual directories which are fully passing
+  // type-checking. Do not remove or disable anything on this list.
+  // Goal: Remove 'QUIET' from all of them.
```

Comment:
Can this be linked via a TODO to an existing issue?

Reply:
We have a new but very in-flux doc at the moment that we plan to make into an I2I after some tweaking. Meanwhile, I'll be actively working in this area in the near future. Would it be alright to link in the issue when it exists?

Reply:
SGTM 👍